### PR TITLE
Releasing 2.5.0

### PR DESCRIPTION
**Pull request by spetratos**
Updated links to latest javadocs in the Extension development guide.

### CODE REVIEW

File: docs/Extension-development-guide.md [hunk 1]
```diff
@@ -103,5 +103,5 @@ When developing a Butterfly extension, it is very helpful to run it from the IDE
 
 Refer to the following javadocs to develop your own Butterfly extension:
 
-1. [butterfly-extensions-api](https://paypal.github.io/butterfly/javadocs/2.4.0/butterfly-extensions-api/index.html): API component to be used by Butterfly extensions when providing their transformation and upgrade templates.
-1. [butterfly-utilities](https://paypal.github.io/butterfly/javadocs/2.4.0/butterfly-utilities/index.html): multiple types of ready-to-use transformation utilities.
+1. [butterfly-extensions-api](https://paypal.github.io/butterfly/javadocs/2.5.0/butterfly-extensions-api/index.html): API component to be used by Butterfly extensions when providing their transformation and upgrade templates.
+1. [butterfly-utilities](https://paypal.github.io/butterfly/javadocs/2..0/butterfly-utilities/index.html): multiple types of ready-to-use transformation utilities.
```
Comment (fabiocarvalho777) [inline docs/Extension-development-guide.md]: @spetratos there is a typo here, you forgot the number 5.

**Comment by spetratos** [comment]
@fabiocarvalho777


Merging change now. Do I need to redo the tag?


Was about to open JIRA for staging repo and build issue. How did you do it last time?


Sp


________________________________

**Comment by fabiocarvalho777** [comment]
@spetratos last time I deploy it from my own laptop running `mvn deploy`. But that won't work for you because you would need Maven credentials in your environment plus my private key to sign the jars. Both, credentials and private key (encrypted), are in Travis CI, and that is why I created that yaml script for releases, so that nobody needs to run `mvn deploy` from a laptop.

**Comment by spetratos** [comment]
Working with sonatype people:


https://issues.sonatype.org/browse/OSSRH-39389?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=477141#comment-477141


Sp


________________________________

**Comment by spetratos** [comment]
@fabiocarvalho777,

<https://github.com/spetratos>

Release 2.5.0. Figured out what the issue was with split repositories.


The badge in the readme.md still shows 2.4.0. Not sure if github caches images.


sp


________________________________

**Pull request merged**